tools/cdk-build-tools/bin/cdk-test.ts

```diff
@@ -40,40 +27,12 @@ async function main() {
     await shell(options.test, defaultShellOptions);
   }
 
-  const testFiles = await unitTestFiles();
-  const useJest = options.jest;
-
-  if (useJest) {
-    if (testFiles.length > 0) {
-      throw new Error(`Jest is enabled, but ${testFiles.length} nodeunit tests were found!: ${testFiles.map(f => f.filename)}`);
```

Comment:
IMO, we should keep this check to detect if any NodeUnit file was created unintentionally. Such tests will be ignored silently now.

Reply:
Good idea, I've added the check back.